tools/health-monitor/README.md

```diff
@@ -0,0 +1,99 @@
+# Blobfuse2 Health Monitor (Preview)
+
+## About
+
+Blobfuse2 Health Monitor is a tool which will help in monitoring the mounts done via Blobfuse2. It supports the following types of monitors:
```

Comment:
We can mention here, there will one instance of health-monitor running per mount (if configured)

Reply:
Added this note.
> **Note:** Health Monitor runs as a separate process where one health monitor process is associated with monitoring one blobfuse2 mounted directory.